crates/bevy_ecs/src/query/filter.rs

```diff
@@ -289,13 +289,14 @@ impl<'w, 's, T: Component> Fetch<'w, 's> for WithoutFetch<T> {
 // SAFETY: no component access or archetype component access
 unsafe impl<T> ReadOnlyFetch for WithoutFetch<T> {}
 
-/// A filter that tests if any of the given filters apply.
+/// A query filter that tests if any of the given filters apply.
 ///
-/// This is useful for example if a system with multiple components in a query only wants to run
-/// when one or more of the components have changed.
+/// Multiple query filters can be nested inside of [Or] query filters as tuples;
+/// this filter will return an entity if any of the nested filters apply.
 ///
-/// The `And` equivalent to this filter is a [`prim@tuple`] testing that all the contained filters
-/// apply instead.
+/// `And` is the default behavior:
+/// adding mutliple filters as a tuple to the second type parameter of [Query](crate::system::Query)
+/// will only return an entity if all of the applied filters are met.
```

Comment:
if this is were how to do `And` is documented, it could use a `#[doc(alias = "and")]`